### PR TITLE
EZP-29355 As a developer, I want to edit the "ezcontent" of a custom tag

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZP-29355
| Versions      | master

Documented `ezcontent` property on FactBox example

### CODE REVIEW

File: docs/guide/extending_online_editor.md [hunk 1]
```diff
@@ -94,6 +100,73 @@ ezrichtext.custom_tags.ezyoutube.attributes.width.label: Width
...
+                        required: true
+                        default_value: 'light'
+                        choices: ['light', 'dark']
+```
+
+Remember to provide your own files for the template and the icon.
+Line 10 points to `ezfactbox.html.twig` template described below.
+Attributes listed below custom tag can be set when adding the tag to a RichText Field.
+
+The configuration requires an `ezfactbox.html.twig` template for the custom tag that will be placed in `/Resources/views/field_type/ezrichtext/custom_tag`:
```
Comment (dew326) [inline docs/guide/extending_online_editor.md]: The path `/Resources/views/field_type/ezrichtext/custom_tag` seems to not be the same as the one provided in the configuration.

File: docs/guide/extending_online_editor.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 The Online Editor is based on [Alloy Editor](https://alloyeditor.com/).
 Refer to [Alloy Editor documentation](https://alloyeditor.com/docs/develop/) to learn how to extend the Online Editor with new elements.
+To learn how to extend the eZ Platform Back Office follow [Extending Admin UI tutorial](../../tutorials/extending_admin_ui/extending_admin_ui).
```
Comment (DominikaK) [inline docs/guide/extending_online_editor.md]: one `/../` to many in the link

File: docs/guide/extending_online_editor.md [hunk 3]
```diff
@@ -12,10 +13,15 @@ Refer to [Alloy Editor documentation](https://alloyeditor.com/docs/develop/) to
 ## Custom tags
 
 Custom tags enable you to add more features to the Rich Text editor beyond the built-in ones.
+They are configured under the `ezrichtext` key. 
 
-Custom tags are configured under the `ezrichtext` key (here on the example of a YouTube tag):
+If you want to learn how to apply them to your installation follow [Creating a custom tag tutorial](../../tutorials/extending_admin_ui/6_adding_a_custom_tag).
```
Comment (DominikaK) [inline docs/guide/extending_online_editor.md]: same here

File: docs/guide/extending_online_editor.md [hunk 4]
```diff
@@ -75,15 +81,15 @@ The configuration requires a Twig template for the custom tag:
 
     Remember that if an attribute is not required, you need to check if it is defined in the template, for example:
 
-    ``` html+twig
+    ```twig
```
Comment (DominikaK) [inline docs/guide/extending_online_editor.md]: Why the change here? It stops html parts of the code block being highlighted.

File: docs/guide/extending_online_editor.md [hunk 5]
```diff
@@ -94,6 +100,73 @@ ezrichtext.custom_tags.ezyoutube.attributes.width.label: Width
 ezrichtext.custom_tags.ezyoutube.attributes.align.label: 'Align'
 ```
 
+### FactBox tag
+
+FactBox tag is a good example for showcasing possibilities of ezcontent property.
```
Comment (DominikaK) [inline docs/guide/extending_online_editor.md]: `ezcontent` in backslashes here and below

File: docs/guide/extending_online_editor.md [hunk 6]
```diff
@@ -94,6 +100,73 @@ ezrichtext.custom_tags.ezyoutube.attributes.width.label: Width
...
+Each custom tag has an ezcontent property, that contains the tag's main content.
```
Comment (DominikaK) [inline docs/guide/extending_online_editor.md]: no comma before that

File: docs/guide/extending_online_editor.md [hunk 7]
```diff
@@ -94,6 +100,73 @@ ezrichtext.custom_tags.ezyoutube.attributes.width.label: Width
...
+                    style:
+                        type: 'choice'
+                        required: true
+                        default_value: 'light'
+                        choices: ['light', 'dark']
+```
+
+Remember to provide your own files for the template and the icon.
+Line 10 points to `ezfactbox.html.twig` template described below.
+Attributes listed below custom tag can be set when adding the tag to a RichText Field.
```
Comment (DominikaK) [inline docs/guide/extending_online_editor.md]: "below `custom_tag`" or "below the custom tag"

File: docs/guide/extending_online_editor.md [hunk 8]
```diff
@@ -12,10 +13,15 @@ Refer to [Alloy Editor documentation](https://alloyeditor.com/docs/develop/) to
 ## Custom tags
 
 Custom tags enable you to add more features to the Rich Text editor beyond the built-in ones.
+They are configured under the `ezrichtext` key. 
 
-Custom tags are configured under the `ezrichtext` key (here on the example of a YouTube tag):
+If you want to learn how to apply them to your installation follow [Creating a custom tag tutorial](../../tutorials/extending_admin_ui/6_adding_a_custom_tag).
+ 
+### YouTube tag
```
Comment (DominikaK) [inline docs/guide/extending_online_editor.md]: Maybe "Example: YouTube tag" and the same below with factbox? So it's clear it just an example you can adapt to your needs.
